spec/index.bs

```diff
@@ -6034,12 +6066,20 @@ type and each [=binding type=] has an associated [=internal usage=], given by th
                 : |entry|.{{GPUBindGroupLayoutEntry/buffer}}?.{{GPUBufferBindingLayout/type}}
                     is {{GPUBufferBindingType/"storage"}} or {{GPUBufferBindingType/"read-only-storage"}}
                 :: Consider 1 {{supported limits/maxStorageBuffersPerShaderStage}} slot to be used.
+                    : If |stage| is {{GPUShaderStage/VERTEX}},
+                    :: Consider 1 {{supported limits/maxStorageBuffersInVertexStage}} slot to be used.
+                    : Else if |stage| is {{GPUShaderStage/FRAGMENT}},
+                    :: Consider 1 {{supported limits/maxStorageBuffersInFragmentStage}} slot to be used.
```

Comment:
This block should have a `<dl class=switch>` around it. Also probably should be written as a proper switch.

Combining both suggestions:

```
: entry.type is storage or read-only-storage
:: If |stage| is:
    <dl class=switch>
        : VERTEX
        :: ...
        : FRAGMENT
        :: ...
        : COMPUTE
        :: ...

            Note: For other stages, this limit is covered by the stage-specific limits
            (due to "[=a new device|limit normalization=]").
    </dl>
```